README.md

```diff
@@ -59,6 +59,24 @@ A simple helper to perform async application initialization for the generic host
     }
     ```
 
+    you can also pass a `CancellationToken` in order to propagate notifications to cancel the initialization if needed.<br />
```

Comment:
- First letter in the sentence should be capitalized
- No need for `<br />` in Markdown, just leave a blank line to introduce a new paragraph, or two spaces at the end of the line for a simple line break.